04-onion-routing.md

```diff
@@ -259,6 +259,7 @@ The reader:
     - MUST return an error if:
       - incoming `amount_msat` < `amt_to_forward`.
       - incoming `cltv_expiry` < `outgoing_cltv_value`.
+      - incoming `cltv_expiry` < `current_block_height` + `min_final_cltv_expiry_delta`.
```

Comment:
With this added, is the other condition `incoming cltv_expiry` < `outgoing_cltv_value` still useful?

Reply:
I think we need both: this one is for the case where the sender doesn't use any "dummy hop" to help hide the recipient, but ideally senders should use an `outgoing_cltv_value` that is much higher than `current_block_height` + `min_final_cltv_expiry_delta` to make it look like there are other hops afterwards.

Reply:
Ah yes, that makes sense. I think in lnd, we just add three blocks in case blocks are found while the htlc is in flight.